appserver/tests/payara-samples/pom.xml

```diff
@@ -88,7 +88,7 @@
         <profile>
             <!--
             This profile will install a Payara server and start up the server per sample.
-            The Payara version that's used can be set via the payara.version property.
+            The Payara version that's used can be set via the project.version property.
```

Comment:
I'm not sure `project.version` is user-modifiable...

Reply:
Good spot! Must've slipped under my radar, will fix now